code/modules/shuttle/navigation_computer.dm

```diff
@@ -29,17 +26,13 @@
 	. = ..()
 	GLOB.navigation_computers += src
 	if(access_station)
-		jumpto_ports += list("nav_z1" = 1)
```

Comment:
This is code cleanup for these, since half of these zlevels dont even exist anymore, and having them hardcoded is bad.